src/modules/client.ts

```diff
@@ -132,13 +134,16 @@ export default class Client {
               ? this.fetchOptions()
               : this.fetchOptions;
           // Fetch data
-          fetch(this.url, {
+          fetchDedupe(this.url, {
             body,
             headers: { 'Content-Type': 'application/json' },
             method: 'POST',
             ...fetchOptions,
+          }, {
+            requestKey: hash,
+            responseType: 'json',
+            dedupe: dedupeRequest
           })
-            .then(res => res.json())
             .then(response => {
               if (response.data) {
```

Comment:
Amusingly, this line needs to become `response.data.data` 🙂 

The `response` argument here isn't a Response object, but is instead the parsed _body_ of a response that happens to have a `data` key. `fetchDedupe` returns you the actual response body with a new attribute, `data`, that is the parsed body. Which is how we end up with `response.data.data`. Awesome